pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -45,6 +47,10 @@ var (
 	baseExtensionState testhelpers.ExtensionState
 )
 
+func init() {
+	tput.InitWatcher(time.Second)
```

Comment:
why not tput.InitWatcher(0) here? Can you add comment?

Reply:
Its to show throughput in upgrade tests, but yes we can disable it here.